src/Command/Setup.php

```diff
@@ -139,6 +140,10 @@ private function getEnvName(InputInterface $input, string $cwd): EnvName
             return EnvName::get($templateName);
         }
 
+        if ($envName = $this->getEnvNameFromChoiceList($io)) {
+            return EnvName::get($envName);
+        }
```

Comment:
couldn't it be `return $this->getEnvNameFromChoiceList();`, removing the `CUSTOM` fallback? Actually the fallback can't be reached since the question will be repeated until a valid answer is given